pkg/cmd/capten.go

```diff
@@ -41,7 +41,9 @@ var appsCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		configPath, _ := cmd.Flags().GetString("config")
 		kubeCfgPath, _ := cmd.Flags().GetString("kubeconfig")
-		if err := util.OsExec("bash", "./generate.sh"); err != nil {
+
+		// generating certificates
+		if err := cert.GenerateCerts(); err != nil {
 			logrus.Errorf("failed to generate certificate %v", err)
```

Comment:
pass with dir path where certs to be generated, and cert-config data

Reply:
cert-config data can be read from capten.yaml in GenerateCerts method, but capten.yaml of cert to be passed